plan/README.md

```diff
@@ -0,0 +1,71 @@
+# Site Plan - MillennialMoney
+
+## Overview
+
+* The mission of *MillennialMoney.com* is to offer young adults and adolescents advice on their finances. Millennialmoney will be designed around giving easy to understand advice to young kids and young adults on money-saving techniques and economical lifestyle tips. 
+
+* Money is everywhere. You can’t escape it. Unfortunately, personal finance is not a required subject in most high schools or colleges. The lack of basic financial education leaves many young adults clueless about how to manage their money, applying for credit, and how to get or stay out of debt. Many people are left at a disadvantage when entering the real world  
+
+* This website will remain static in nature and will be built from the bare bone up. The website will consist of several pages that each have some kind of of simple and easy to understand advice about how to take care of your personal finances.  
+
```

Comment:
Excellent overview! Instead of using bullet points here, just write in paragraph form.